src/features/Instructor/AddEvent/index.jsx

```diff
@@ -0,0 +1,173 @@
+      fontSize: '1rem',
+      fontWeight: '700',
+    }),
+  };
+
+  useEffect(() => {
+    if (!isOpen) {
+      setEventData(initialStateModal);
+    }
+  }, [isOpen]);
+
+  return (
+    <ModalDialog
+      title="New event"
+      isOpen={isOpen}
+      onClose={onClose}
+      hasCloseButton
+    >
+      <ModalDialog.Header>
+        <ModalDialog.Title>New Event</ModalDialog.Title>
+      </ModalDialog.Header>
+      <ModalDialog.Body>
+        <Form>
+          <Form.Row className="col-12 px-0">
+            <Form.Group>
+              <Form.RadioSet
+                name="event"
+                className="d-flex align-items-end justify-content-between col-12 px-1"
+                onChange={e => setEventData({ ...eventData, type: e.target.value })}
+                isInline
+                value={eventData.type}
+              >
+                {typeEventOptions.map(option => (
+                  <Form.Radio value={option.value} key={option.value}>{option.label}</Form.Radio>
+                ))}
+              </Form.RadioSet>
+            </Form.Group>
+          </Form.Row>
+          <Form.Row>
+            <Form.Group className="d-flex align-items-center justify-content-between col-12">
+              <Form.Switch
+                checked={eventData.allDay}
+                onChange={e => setEventData({ ...eventData, allDay: e.target.checked })}
+              >
+                All day
+              </Form.Switch>
+              <Select
+                name="repeat"
+                options={repeatOptions}
+                onChange={option => setEventData({ ...eventData, repeat: option })}
+                value={eventData.repeat}
+                styles={customStyles}
+              />
+            </Form.Group>
+          </Form.Row>
+          <Form.Row>
+            <Form.Group className="d-flex align-items-end justify-content-between col-12">
+              <Form.Control
+                type="date"
+                placeholder="Start date"
+                floatingLabel="Start date"
+                className="my-1 mr-0"
+                name="start_date"
+                required
+                value={eventData.startDate}
+                onChange={e => setEventData({ ...eventData, startDate: e.target.value })}
+              />
+            </Form.Group>
+            <Form.Group className="d-flex align-items-end justify-content-between col-12">
+              <Form.Control
+                type="date"
+                placeholder="End date"
+                floatingLabel="End date"
+                className="my-1 mr-0"
+                name="end_date"
+                required
+                value={eventData.endDate}
+                onChange={e => setEventData({ ...eventData, endDate: e.target.value })}
+              />
+            </Form.Group>
+          </Form.Row>
+          {eventData.allDay === false && (
+          <Form.Row>
+            <Form.Group as={Col}>
+              <Form.Control
```

Comment:
Is not possible to change the icon for an arrow like the design?

Reply:
To override the Icon I have to install the library react- select or to make that in the library, which is a more effort for something doesn't change the user experience. Remember the wirefrime is a guide for the design.